build.gradle

```diff
@@ -40,11 +40,15 @@ repositories {
 
 ext {
     isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+    buildVersionQualifier = System.getProperty("build.version_qualifier")
 }
 
 allprojects {
     group 'org.opensearch.commons'
-    version = opensearch_version - '-SNAPSHOT' + '.0'
+    version = opensearch_version.tokenize('-')[0] + '.0'
```

Comment:
do we really need to tokenize opensearch version ?

Reply:
It's things like `1.2.3-alpha1-SNAPSHOT`, so we just need the first part left of the `-` or the entire thing if it's 1.2.3. Do you know a better way to do it?